instagram-app/src/components/PostContainer/CommentSection/CommentSection.js

```diff
@@ -0,0 +1,86 @@
+import React from 'react';
+import Likes from './Likes/Likes';
+import PostedComments from './PostedComments/PostedComments';
+import AddComment from './AddComment/AddComment';
+import PropTypes from 'prop-types';
+import { CommentSectionDiv } from '../CommentSection/CommentSectionStyles'
+
+class CommentSection extends React.Component {
+    constructor(props) {
+        super(props);
+        this.state = {
+            comments: this.props.comments,
+            currentComment: {username: this.props.comments.username, text: this.props.comments.text},
+            likesCounter: this.props.likes,
+            click: true,
+            showForm: false
+        }
+    }
+
+    handleInput = e => {
+        this.setState({
+            currentComment: {...this.state.currentComment, [e.target.name]: e.target.value }
+        });
+    }
+
+    toggleClick = () => {
+        this.setState({ click: !this.state.click })
+        if (this.state.click === true) {
+            this.setState({ likesCounter: this.state.likesCounter + 1});
+        } else {
+            this.setState({ likesCounter: this.state.likesCounter - 1});
+        }
+    }
```

Comment:
I love that you got the logic to increment and decrement on click! Well done implementing a true part of the Instagram interface.